components/Footer.vue

```diff
@@ -4,29 +4,38 @@
       <p class="footer__copyright">
         Copyright @{{ currentYear }} All rights reserved
       </p>
+      <div class="footer__icons">
+        <a href="#"><img src="../assets/IconFacebook.svg" /></a>
+        <a href="#"><img src="../assets/IconInstagram.svg" /></a>
+        <a href="#"><img src="../assets/IconLinkedin.svg" /></a>
+        <a href="#"><img src="../assets/IconTwitter.svg" /></a>
+        <a href="#"><img src="../assets/IconYoutube.svg" /></a>
+      </div>
```

Comment:
@mishschmid 
In the beginning, I wanted to configure nuxt 3 to add svgs as vue Component, sth like <Icon Facebook />

By adding this [dependency](https://www.npmjs.com/package/@nuxtjs/svg) and following what is under vue-svg-loader